code/game/objects/items/weapons/implants/implant_misc.dm

```diff
@@ -48,6 +48,24 @@
 	imp_in.reagents.add_reagent("omnizine", 10)
 	imp_in.reagents.add_reagent("stimulants", 10)
 
+/obj/item/weapon/implant/adrenalin/deathsquad
+	name = "adrenal implant"
+	desc = "Removes all stuns and knockdowns."
+	icon_state = "adrenal"
+	origin_tech = "bluespace=4;biotech=10;combat=10;syndicate=4"
+	uses = 300
+
+/obj/item/weapon/implant/adrenalin/deathsquad/get_data()
+	var/dat = {"<b>Implant Specifications:</b><BR>
+				<b>Name:</b> Cybersun Industries Military Grade Adrenaline Implant<BR>
```

Comment:
Ninja isn't removed. It just got most of its features gutted and the objective removed.
It's even more of a hard counter to DS currently, as thermals don't reveal the ninja anymore and the ninja stars completely bypass the ds armor.